src/main/java/uk/gov/hmcts/reform/translate/model/GetDictionaryResponse.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.translate.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.Map;
+
+@AllArgsConstructor
+@Data
+public class GetDictionaryResponse {
+    private Map<String, String> translations;
+}
```

Comment:
According to the spec It seems that PUT service and GET are sharing the same payload for request / response .  Could we rename GetDictionaryResponse?   or create an superclass with the main Map  attribute so then we can have concrete implementations for Get-response and Put-request ?

Reply:
Will rename to `DictionaryResponse`

Reply:
Fixed in https://github.com/hmcts/ts-translation-service/pull/6/commits/46585f3e1763b52bd2709a72f4a1cfc2f0ebc77b